### DISCUSSION TITLE
Enable import of non-module stylesheets in components

### DISCUSSION THREAD

**Discussion by Badyl1g** [vercel/next.js#36057] [Help]
We have a very big app that uses regular .scss files imported in components.
Currently we use next@9 and we want to upgrade to version 12
As I understand in order to upgrade to the newest version we need to rewrite all our styles to .module.scss which seems nearly impossible looking at the number of components we have
Is there a way to enable import of non-module stylesheets in components so we can slowly rewrite them to modules one at a time?

I'm aware that nowadays it's not recommended to use global styles in components but I'm sure there are lots of people with legacy apps that can't afford to rewrite all stylesheets but would like to use the latest nextjs versions

It would be great to have a flag in next.config.js that allows import of regular styles, it would still be discouraged to do so, but it would help to adopt newest nextjs versions

**Comment by GabenGar** [comment]
[SASS has its own module system](https://sass-lang.com/documentation/modules) so use it have a million files while also bundling them into a single stylesheet in the end.
